bpf/lib/icmp6.h

```diff
@@ -355,7 +355,7 @@ __section_tail(CILIUM_MAP_CALLS, CILIUM_CALL_HANDLE_ICMP6_NS) int tail_icmp6_han
 
 	ret = __icmp6_handle_ns(skb, nh_off);
 	if (IS_ERR(ret))
-		return send_drop_notify_error(skb, ret, TC_ACT_SHOT);
+		return send_drop_notify_error(skb, ret, TC_ACT_SHOT, DIRECTION_EGRESS);
```

Comment:
This can be both ingress and egress. The direction must be passed in from the original callers.

Reply:
I believe this is being called by the tail call `CILIUM_CALL_SEND_ICMP6_ECHO_REPLY` which is only called at `icmp6_send_echo_reply`. I guess we are replying back to case `ICMPV6_ECHO_REQUEST` in `icmp6_handle`. So since we are replying ourselves to the ICMP reply, I can see that this case is `EGRESS`? Which is the `INGRESS` scenario? @joestringer @tgraf

Reply:
done